package.json

```diff
@@ -59,7 +59,7 @@
     "webpack-cli": "^4.10.0"
   },
   "dependencies": {
-    "@eppo/js-client-sdk-common": "^4.5.1"
+    "@eppo/js-client-sdk-common": "^4.5.3"
```

Comment:
Removes the possibility for a client sdk to be stuck on an older version of common even if they're using the latest version of the client sdk